js/framework/AnalyticsConstants.js

```diff
@@ -324,6 +363,54 @@ if (!OO.Analytics.EVENT_DATA)
     this.totalStreamDuration = checkVideoStreamPositionChangedData(totalStreamDuration, "totalStreamDuration", "number");
   }
 
+  /**
+   * @public
+   * @class Analytics.EVENT_DATA#AdPodStartedData
+   * @classdesc Contain information about how many ads are in the ad pod.
+   * @property {number} numberOfAds
+   */
+  EVENT_DATA.AdPodStartedData = function(numberOfAds)
+  {
+    var checkAdPodStartedData = OO._.bind(checkDataType, this, "AdPodStartedData");
+    this.numberOfAds = checkAdPodStartedData(numberOfAds, "numberOfAds", "number");
+  }
+
+  /**
+   * @public
+   * @class Analytics.EVENT_DATA#AdPodEndedData
+   * @classdesc Contain information about the adId of the ad pod.
+   * @property {string} adId
+   */
+  EVENT_DATA.AdPodEndedData = function(adId)
+  {
+    var checkAdPodEndedData = OO._.bind(checkDataType, this, "AdPodEndedData");
+    this.adId = checkAdPodEndedData(adId, "adId", "string");
+  }
+
+  /**
+   * @public
+   * @class Analytics.EVENT_DATA#FullscreenChangedData
+   * @classdesc Contains information about whether the player is entering or exiting fullscreen.
+   * @property {boolean} shouldEnterFullscreen
+   */
+  EVENT_DATA.FullscreenChangedData = function(shouldEnterFullscreen)
```

Comment:
can you make the var something like "ChangingToFullscreen".  I would like to avoid 'should' and 'will' so as not to confuse non-native english speakers. Also, 'FullscreenChangedData' means it already did that thing, so 'should' is not the right word.